playbook/app/pb_kits/playbook/pb_form_group/docs/_form_group_select.jsx

```diff
@@ -32,6 +31,17 @@ const FormGroupSelect = (props) => {
             {...props}
         />
       </FormGroup>
+      <br />
+      <br />
+      <FormGroup>
+        <Select
+            blankSelection="Phone"
+            options={options}
```

Comment:
Looks great @nickamantia ! Minor thing: since you're passing in the same options object here as the select above this, the drop down has all the music related options. This should be the same as the rails one with 'phone', 'mobile', etc.

Reply:
Great catch!! just added a new options array for the phone options. thanks @nidaqg!!!!